Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1833,78 +1835,146 @@ inline void SetImage(CGImageRef image) {
 
 /**
  @Status Caveat
- @Notes Limited bitmap formats available. Decode, shouldInterpolate, intent parameters
- and some byte orders ignored.
- */
+ The following formats are supported GUID_WICPixelFormat32bppPBGRA, GUID_WICPixelFormat32bppPRGBA, GUID_WICPixelFormat32bppBGR and
+ GUID_WICPixelFormat8bppAlpha. Any other format will return a null value.
```

Comment:
@msft-Jeyaram this is not what I meant.  We should specify what *CG* bitmap formats are supported, not what WicBitmap formats.  Ex: 32bppPBGRA == ARGB and so on.